cds-snc/cppd-medical-report
# Updated French (and some English) for iteration 2

# Description
This adds the French prototype.
https://trello.com/c/Vgs2cgtd/370-physicians-can-use-prototype-in-french
Also has some English changes from our initial analysis of our first round of research: https://trello.com/c/dUHjYcH1/364-analyze-medical-professional-ma-sessions

# Test Instructions

Click on 'Francais' on the top right corner, you should see a full French prototype, and no instances of 'false' anywhere.

# Help Requested

Non-engs should be fine to review this.
